ext/standard/php_fopen_wrapper.c

```diff
@@ -215,23 +215,20 @@ php_stream * php_stream_url_wrap_php(php_stream_wrapper *wrapper, const char *pa
 	}
 
 	if (!strcasecmp(path, "input")) {
-		php_stream_input_t *input;
-
+        php_stream_input_t *input;
```

Comment:
See https://github.com/php/php-src/blob/master/CODING_STANDARDS.md#syntax-and-indentation